repos-github.md

```diff
@@ -552,6 +552,7 @@
 - japgolly/univeq
 - jatcwang/doobieroll
 - jatcwang/scalafix-named-params
+- jbwheatley/pact4s
```

Comment:
I know someone who used this exact name, so now they'll need to change it if they OSS it :joy:

Reply:
ahaha, really! Glad I got in first 🙊